ARM.CMSIS.pdsc

```diff
@@ -2174,16 +1308,290 @@ and 8-bit Java bytecodes in Jazelle state.
       <require Cclass="Device" Cgroup="Startup"/>
     </condition>
 
+    <condition id="RTOS RTX ARM ARMv6-M">
```

Comment:
Condition is not really RTOS RTX specific and could be used also elsewhere.
Better name would be "ARMCC ARMv6-M LE". That would reflect exactly the condition contents.
Also it would make sense to move this whole section outside RTX specific conditions.

Reply:
I added RTOS RTX because currently these conditions are only used for RTX.

Reply:
Yes, there are currently only used for RTX (and might not be used for anything else). But still the condition name is misleading.

Reply:
Okay, I'll remove all the "RTOS RTX" prefixes.